rllib/models/modelv2.py

```diff
@@ -318,6 +319,24 @@ def is_time_major(self) -> bool:
         """
         return self.time_major is True
 
+    @PublicAPI
+    def update_view_requirements_from_init_state(self):
```

Comment:
This method doesn't seem to be used, can we remove it? The semantics are also kind of hard to understand, I don't think we want to expose such a public API.